IzvorniKod/frontend/src/components/SearchComponent/SearchComponent.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { Input } from "antd";
+import {
+  PaperClipIcon,
+  PaperAirplaneIcon,
+  CpuChipIcon,
+} from "@heroicons/react/24/solid";
+import { cn } from "../../utils/cn.util";
+
+const SearchComponent: React.FC = () => {
+  const [answerContent, setAnswerContent] = useState("");
```

Comment:
mozes slobodno uklonit ovaj React.FC tip jer nam ne treba. Isto tako moramo imat ulazni parametar u komponentu - `postContent: string;` sto je zapravo sadrzaj pitanja koji cemo slat na BE kada se generira odgovor sa LLM-om